### PR TITLE
Regenerate Mvc functional test baselines

### PR DESCRIPTION
**Pull request by dougbu**
- whitespace likely changed due to a recent Razor change (but there've been so many!)

### CODE REVIEW

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (dougbu) [inline src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html]: This may actually cause a new failure because response content is trimmed before comparing with the baseline. I'm testing locally to confirm. If that's true, I'll update the `GENERATE_BASELINES` part of the test to stabilize things.

**Comment by dougbu** [comment]
:up::date: to also avoid failures described in #4417 
- local src/Mvc builds are now clean

**Comment by dougbu** [comment]
Bump.

This fixes a few of the current build breaks in this repo. Let's get it in.

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs [hunk 2]
```diff
@@ -56,7 +56,7 @@ public async Task Page_SimpleForms_RenderAntiforgery()
             ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
 #else
             expectedContent = string.Format(expectedContent, forgeryToken);
-            Assert.Equal(expectedContent, responseContent.Trim(), ignoreLineEndingDifferences: true);
+            Assert.Equal(expectedContent, responseContent, ignoreLineEndingDifferences: true);
```
Comment (ajaybhargavb) [inline src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs]: I don't understand why this change was necessary

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.ProductListUsingTagHelpers.html [hunk 3]
```diff
@@ -58,7 +58,6 @@
     <textarea rows="4" cols="50" class="product" id="z2__Description" name="[2].Description">
 Product_2 description</textarea>
 </div>
-
```
Comment (ajaybhargavb) [inline src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.ProductListUsingTagHelpers.html]: I just finished investigating it. And this change is by design and is a bug fix. It's fine to get this in

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html (hunk 1 above)
Comment (ajaybhargavb) [inline src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html]: Not sure what happened here. Doesn't seem to be related to my Razor changes. @dougbu is this a result of removing the `Trim()` above?

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs (hunk 2 above)
Comment (dougbu) [reply src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs]: This is directly related to the build break. If we trim the response and the expected text contains a leading newline, the test fails.

As I said in the description, trimming the responses is wasteful and incorrect. No reason for this in any of the functional tests using `GENERATE_BASELINES`.

File: src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html (hunk 1 above)
Comment (dougbu) [reply src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html]: No, this is the result of re-running with `GENERATE_BASELINES`. This file now contains the correct baseline text.

Comment (ajaybhargavb) [reply src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html]: I ask because this wasn't a test failure when I ran locally

**Review by NTaylorMullen — APPROVED** [review]
